github-event-service/src/push.ts

```diff
@@ -0,0 +1,59 @@
+import * as fetch from 'node-fetch';
+import {safeLoad as parseYaml} from 'js-yaml';
+
+import {getFirebaseInstance} from './firebase-init';
+
+/** The settings object for the branch manager. */
+export interface BranchManagerSettings {
+  enabled: boolean;
+  branches: Array<{
+    branch: string;
+    label: string;
+  }>;
+};
+
+/** 
+ * Limited implementation of the Github push webhook event from
+ * https://developer.github.com/webhooks/#payloads
+ */
+export interface GithubPushEvent { 
+  ref: string;
+  repository: {
+    id: number;
+    full_name: string;
+  }
+};
+
+/** The Firebase app. */
+const firestore = getFirebaseInstance().firestore();
+
+/** Handle webhook events from github of the type, `push`. */
+export function handlePushEvent(event: GithubPushEvent) {
+  if (event.ref === 'refs/tags/master') {
```

Comment:
Make a function like `isMasterBranch(ref)`?

Reply:
I feel like it might be overkill to create method just to perform this one equality check.

Reply:
It's not so much about the amount of code, but making the code here read more like a sentence